src/fileprovider.cs

```diff
@@ -1,4 +1,4 @@
-﻿﻿﻿//
```

Comment:
Is this intentional?

Reply:
Is an editor either adding or removing a BOM (https://en.wikipedia.org/wiki/Byte_order_mark)